solidity/random-beacon/test/RandomBeaconGovernance.test.ts

```diff
@@ -12,96 +12,25 @@ import type {
 } from "../typechain"
 
 const { createSnapshot, restoreSnapshot } = helpers.snapshot
-const { to1e18 } = helpers.number
 
 const governanceDelay = 604800 // 1 week
 
-const initialRelayEntrySoftTimeout = 1280
-const initialRelayEntryHardTimeout = 5760
-const initialCallbackGasLimit = 56000
-const initialGroupCreationFrequency = 2
-const initialGroupLifeTime = 259200
-const initialDkgResultChallengePeriodLength = 11520
-const initialDkgResultSubmissionTimeout = 1280
-const initialDkgSubmitterPrecedencePeriodLength = 20
-const initialRelayEntrySubmissionFailureSlashingAmount = to1e18(400)
-const initialMaliciousDkgResultSlashingAmount = to1e18(400)
-const initialUnauthorizedSigningSlashingAmount = to1e18(400)
-const initialSortitionPoolRewardsBanDuration = 1209600
-const initialRelayEntryTimeoutNotificationRewardMultiplier = 100
-const initialUnauthorizedSignatureNotificationRewardMultiplier = 100
-const initialMinimumAuthorization = to1e18(40000)
-const initialAuthorizationDecreaseDelay = 3888000
-const initialAuthorizationDecreaseChangePeriod = 3888000
-const initialDkgMaliciousResultNotificationRewardMultiplier = 100
 const initialDkgResultSubmissionGas = 235000
-const initialDkgResultApprovalGasOffset = 43500
+const initialDkgResultApprovalGasOffset = 41500
 const initialNotifyOperatorInactivityGasOffset = 54500
-const initialRelayEntrySubmissionGasOffset = 11500
+const initialRelayEntrySubmissionGasOffset = 11250
```

Comment:
Why did we not move the remaining 5 parameters to `const params` in `test/fixtures/index.ts`?

Reply:
All the other values but these 5 were used in other tests. If you think they also belong there, I moved them [be3a783](https://github.com/keep-network/keep-core/pull/3018/commits/be3a783a2c6231286663e3103aad9b6130a30c9e)